aggregator/src/main/scala/weco/concepts/aggregator/ConceptExtractor.scala

```diff
@@ -13,7 +13,7 @@ object ConceptExtractor extends Logging {
   def apply(jsonString: String): Seq[CatalogueConcept] = {
     val jsonObj = ujson.read(jsonString)
     val concepts = allConcepts(List(jsonObj), Nil).toList
-      .distinctBy(_.identifier)
+      .distinctBy({ concept => (concept.identifier, concept.ontologyType) })
```

Comment:
Worth putting the tuple as a `val` on `CatalogueConcept` (or something like that) so behaviour is guaranteed the be consistent across `Extractor` and `Aggregator`?